requirements.txt

```diff
@@ -1,5 +1,6 @@
 enum34==1.1.6
 Jinja2==2.8
-lxml==3.6.4
+lxml #==3.6.4
```

Comment:
Sorry, thought I'd commented on this before -- what does `#=` mean in this context? I looked at the docs and didn't see a specific mention of it. If it's a comment, should we just remove `==3.6.4`?